aria/orchestrator/execution_plugin/ctx_proxy/server.py

```diff
@@ -150,17 +146,29 @@ def __exit__(self, *args, **kwargs):
         self.close()
 
 
-def _process_ctx_request(ctx, args): # pylint: disable=too-many-branches,too-many-statements
-    current = ctx
-    index = 0
+class ProcessingError(RuntimeError):
```

Comment:
consider `CtxArgProcessingError`, or any other name since `ProcessingError` makes it sound like its a process related issue.